libbeat/processors/registry.go

```diff
@@ -52,7 +52,7 @@ type Constructor func(config *common.Config) (Processor, error)
 var registry = NewNamespace()
 
 func RegisterPlugin(name string, constructor Constructor) {
-	logp.Debug("processors", "Register plugin %s", name)
+	logp.L().Named(logName).Debugf("Register plugin %s", name)
```

Comment:
I always get forgot about by this change `logp.L()` in my code, we will need to audit the code to remove any direct calls to logp.